tldw_Server_API/app/core/MCP_unified/tests/test_filesystem_module.py

```
@@ -0,0 +1,118 @@
+    resolver = _FakeWorkspaceRootResolver(tmp_path)
+    module = FilesystemModule(
+        ModuleConfig(
+            name="filesystem",
+            settings={"workspace_root_resolver": resolver},
+        )
+    )
+    await module.initialize()
+    return module, resolver
+
+
+def _make_context(tmp_path: Path, *, cwd: str = ".") -> RequestContext:
```

Ruff (0.15.7)

[warning] Unused function argument: `tmp_path`

(ARG001)